financial-connections-example/build.gradle

```diff
@@ -27,6 +27,10 @@ android {
                 groups = "financial-connections"
             }
         }
+        release {
+            minifyEnabled true
```

Comment:
You shouldn't need this, it should already be handled by https://github.com/stripe/stripe-android/blob/9e65354966ceef2d35c1f71afad9d00e5aa5d16c/build-configuration/android-application.gradle#L40

Reply:
https://github.com/stripe/stripe-android/pull/6813/commits/a187372b29e3fe1a5585cc0a37c6050f0f3a9190